plugins/attachment-resources/src/utils.ts

```diff
@@ -99,7 +101,7 @@ export function getType (type: string): 'image' | 'text' | 'json' | 'video' | 'a
   if (type.startsWith('video/')) {
     return 'video'
   }
-  if (type.includes('application/pdf')) {
+  if (type === 'application/pdf') {
```

Comment:
Please revert this change. I think `includes` was used intentionally, because type can be something like `application/pdf;charset=utf-8`